tests/unit/tasks/apiRequests/get.ts

```diff
@@ -0,0 +1,32 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Previously I had tested the pagination methods incredibly deeply for the loginAttempts, we agreed it would be unnecessary to do it again as they're just copy paste into the interfaces.

Thus I have just tested the construction / targeting of the correct urls here.